x/evm/keeper/grpc_query_test.go

```diff
@@ -983,13 +935,9 @@ func getDefaultTraceTxRequest(unitNetwork network.Network) types.QueryTraceTxReq
 	}
 }
 
-func (suite *EvmKeeperTestSuite) TestTraceTx() {
-	keyring := testkeyring.New(2)
-	unitNetwork := network.NewUnitTestNetwork(
-		network.WithPreFundedAccounts(keyring.GetAllAccAddrs()...),
-	)
-	grcpHandler := grpc.NewIntegrationHandler(unitNetwork)
-	txFactory := factory.New(unitNetwork, grcpHandler)
+func (suite *KeeperTestSuite) TestTraceTx() {
+	suite.enableFeemarket = true
+	suite.SetupTest()
```

Comment:
Nit: add a defer function to reset the `enableFeemarket` flag after this test
```suggestion
	suite.SetupTest()
	defer func() {suite.enableFeemarket = false }()
```